packages/app-store/slackapp/lib/showTodayMessage.ts

```diff
@@ -0,0 +1,50 @@
+import { BookingStatus } from "@prisma/client";
+import { NextApiRequest, NextApiResponse } from "next";
+
+import { WhereCredsEqualsId } from "./WhereCredsEqualsID";
+import { NoUserMessage, TodayMessage } from "./views";
+
+export default async function showCreateEventMessage(req: NextApiRequest, res: NextApiResponse) {
+  const body = req.body;
+
+  const foundUser = await prisma.credential.findFirst({
+    ...WhereCredsEqualsId(body.user_id),
+    include: {
+      user: {
+        select: {
+          id: true,
+          email: true,
+        },
+      },
+    },
+  });
+
+  if (!foundUser) res.status(200).json(NoUserMessage);
+  const bookings = await prisma.booking.findMany({
+    where: {
+      OR: [
+        {
+          userId: foundUser?.userId,
+        },
+        {
+          attendees: {
+            some: {
+              email: foundUser?.user?.email,
+            },
+          },
+        },
+      ],
+      AND: [
+        {
+          endTime: { gte: new Date() },
+          AND: [
```

Comment:
Do we include bookings that require opt in and `confirmed: false`?

Reply:
Currently no - as from what i can tell we are fetching the data the exact same way as bookings/upcoming